packages/svelte/src/internal/client/render.js

```diff
@@ -1561,11 +1567,12 @@ export function element(anchor_node, tag_fn, render_fn, is_svg = false) {
 	// Managed effect
 	const render_effect_signal = render_effect(
 		() => {
+			const ns = namespace ?? tag === 'svg' ? namespace_svg : null;
```

Comment:
This looks fishy. The `namespace` is a `string`, but the fallback is a boolean, and the namespace value is never used, instead it's either the `namespace_svg` value or `null`. Either the `namespace` is trusted and the fallback is fo use `namespace_svg` if the tag is `svg`, or the `namespace` parameter becomes `boolean | null` or sth similar to distinguish the three cases of "known yes/no, unknown".

Reply:
it just needs some parens

```suggestion
			const ns = namespace ?? (tag === 'svg' ? namespace_svg : null);
```

though making that change surfaced some type errors, so I just pushed a drive-by fix